src/main/groovy/com/twcable/grabbit/jcr/ProtoPropertyDecorator.groovy

```diff
@@ -70,6 +71,12 @@ class ProtoPropertyDecorator {
                 log.warn "WARNING!  Property ${name} will not be written to ${node.name}!  There was a problem when writing value type ${PropertyType.nameFromValue(type)}${multiple ? '[]' : ''} to existing node with same type, due to a ValueFormatException, and we were unable to recover"
             }
         }
+        catch (VersionException ex){
```

Comment:
We need to be proactive and check for this case up front; especially since this isn't an exception case. Intercept, and act. See some of my comments on your previous PR.